spec/requests/app_spec.rb

```diff
@@ -21,4 +21,15 @@
       expect(parsed_response).to eq(data_object_for(verse))
     end
   end
+
+  describe 'GET /devotionals' do
+    let!(:devotionals) { create_list(:devotional, 6) }
+
+    it 'returns a list of devotionals' do
+      get '/devotionals', {}, headers
```

Comment:
Is this route only for authenticated users or is this public? What would happen if an unauthenticated user tried to get here?

Reply:
I would say we only want this route to be accessible to authenticated users since it is behind the login screen. If an unauthenticated user did get here we would want to deny them access.

Reply:
Yep, I agree. Let's have some tests for this case.

Reply:
Okay cool! Will probably need the same for /home right?

Reply:
Yep, but if you wanted to add that in a small followup PR that's cool too. Up to you.